torch/nn/parallel/distributed.py

```diff
@@ -478,6 +478,21 @@ class DistributedDataParallel(Module, Joinable):
                       gradients. If hitting such errors, please fix it by
                       referring to the :meth:`~torch.optim.Optimizer.zero_grad`
                       function in ``torch/optim/optimizer.py`` as a solution.
+        static_graph (bool): When set to ``True``, DDP knows the trained graph is
+                     static. Static graph means 1) the set of used and unused
+                     parameters will not change during the whole training loop; in
+                     this case, it does not matter whether users set
+                     find_unsued_parameters = true or not. 2) how the graph is trained
+                     will not change during the whole training loop (meaning there is
+                     no control flow depending on iterations). When graph is set to be
+                     static, DDP will support cases that can not be supported in the past:
+                     1) reentrant backwards 2) activation checkpointing multiple times
+                     3) activation checkpointing with find_unused_parameters = true.
```

Comment:
Since we earlier in the doc say that find_Unused_params=True doesn't matter when static graph is set, maybe we can clarify this as "activation checkpointing when model has unused parameters".